sshcode.go

```diff
@@ -505,6 +520,7 @@ func rsync(src string, dest string, sshFlags string, excludePaths ...string) err
 		// locally in order to properly delete an extension.
 		"--delete",
 		"--copy-unsafe-links",
+		"-zz",
```

Comment:
:+1: Nice addition

Reply:
Don't even remember where I got that, it compresses as it is sending, in theory making transfer faster. 

It fixed SOMETHING, I just forgot what,